build/upgrade-deps.ps1

```diff
@@ -0,0 +1,64 @@
+﻿#!/usr/bin/env powershell
```

Comment:
rather than depending on XML parsing and a relatively complex logic being implemented twice, would it be 
easier to just hardcode  the latest Grpc version "1.19.0-dev"  and the prefix for downloading the nuget packages?
(it's always the same prefix for all nugets from the same build).
Besides thing being just simpler that way, I think it's better to rely on a fixed version of the nupkgs (and update the download prefix manually if needed) rather than automatically downloading the latest version.

Reply:
I think I'm going to change the implementation a little since I wanted to add scripts to remove the manual step. I think I'll split this into two scripts one to update the version and one to download the packages. Most builds will run the download only and will not need the xml dependency. We will run the update version tool manually when we need to ingest new packages.

Reply:
@JamesNK This would also allow us to control when to ingest new changes.